photon-icons/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "photon-icons",
+  "version": "0.0.1",
+  "description": "Icons for Firefox and related properties.",
+  "scripts": {
+    "build": "node build.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/FirefoxUX/design-tokens.git"
+  },
+  "keywords": [
+    "firefox",
+    "icons",
+    "design",
+    "tokens",
+    "json"
+  ],
+  "author": "Blake Winton <bwinton@latte.ca>",
```

Comment:
Out of curiosity, why there isn't an `@mozilla.org` email?

Reply:
No reason. 🙂